content/en/news/releases/1.16.x/announcing-1.16/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Announcing Istio 1.16
+linktitle: 1.16
+subtitle: Major Update
+description: Istio 1.16 release announcement.
+publishdate: 2022-11-15
+release: 1.16.0
+skip_list: true
+aliases:
+- /news/announcing-1.16
+- /news/announcing-1.16.0
+---
+
+We are pleased to announce the release of Istio 1.16!
+
+{{< relnote >}}
+
+This is the fourth Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.16.0 published. Special thanks are due to the release managers Daniel Hawton from Solo.io, Ziyang Xiao from Intel, and Tong Li from IBM. As always, our gratitude goes to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
+
+{{< tip >}}
+Istio 1.16.0 is officially supported on Kubernetes versions `1.22` to `1.25`.
+{{< /tip >}}
+
+## What's new
+
+Here are some of the highlights of the release:
+
+### MAGLEV Load Balancing Support
+
+We have added support for the MAGLEV load balancing algorithm.
```

Comment:
Assuming this is configured via DR?  Would be good to link to envoy doc as well: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/load_balancers#maglev

Reply:
Thanks, Lin. Adding.